teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<p>Note also that drugs can also be administered via alternative routes, i.e. inhalation, skin penetration and injection.  </p><p>-&gt; Note <s>also</s> that drugs can also be administered via alternative routes, i.e. inhalation, skin penetration and injection.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>